profile/profile_paddleocr.py

```diff
@@ -0,0 +1,120 @@
+from paddleocr import PaddleOCR
+from dataclasses import dataclass, asdict
+
+from PIL.Image import Image as ImageType
```

Comment:
This is a bit confusing. Why do we need to rename `Image`?

Reply:
cleaned up